src/services/featurehelper.js

```diff
@@ -197,61 +197,88 @@ ngeo.FeatureHelper.prototype.getStyle = function(feature) {
 
 /**
  * @param {!ol.Feature} feature Feature with linestring geometry.
- * @return {!ol.style.Style} Style.
+ * @return {!Array.<!ol.style.Style>} Style.
  * @private
  */
 ngeo.FeatureHelper.prototype.getLineStringStyle_ = function(feature) {
 
   const strokeWidth = this.getStrokeProperty(feature);
+  const showLabel = this.getShowLabelProperty(feature);
   const showMeasure = this.getShowMeasureProperty(feature);
   const color = this.getRGBAColorProperty(feature);
 
-  const options = {
+  const styles = [new ol.style.Style({
     stroke: new ol.style.Stroke({
       color: color,
       width: strokeWidth
     })
-  };
-
+  })];
+  let offset = 0;
   if (showMeasure) {
-    options.text = this.createTextStyle_({
-      text: this.getMeasure(feature)
-    });
+    offset += 25;
+    styles.push(new ol.style.Style({
+      text: this.createTextStyle_({
+        text: this.getMeasure(feature),
+        offsetY: -(offset / 2 + 4)
+      })
+    }));
+  }
+  if (showLabel) {
+    offset += 25;
+    styles.push(new ol.style.Style({
+      text: this.createTextStyle_({
+        text: this.getNameProperty(feature),
+        offsetY: -(offset / 2 + 4)
+      })
+    }));
```

Comment:
@fredj is this offset calculation correct?

Reply:
no idea

Reply:
This 25 looks to be a magic number, shouln't we use multiline from OpenLayers: https://github.com/openlayers/openlayers/pull/4512/?